swe/solver.py

```diff
@@ -42,7 +42,6 @@ def set_fields(self, adapted=False):
         self.fields = {}
         self.fields['viscosity'] = self.op.set_viscosity(self.P1)
         self.fields['diffusivity'] = self.op.set_diffusivity(self.P1)
-        self.fields['bathymetry'] = self.op.set_bathymetry(self.P1DG)
```

Comment:
Why are you removing bathymetry from the fields dict? It is needed in all of my test cases.